src/states/Menu.js

```diff
@@ -1,20 +1,126 @@
+    this.drawMain();
   },
 
   update: function() {
-
+    this.backDropFilter.update(window.game.input.activePointer);
   },
 
   render: function() {
+    
+  },
 
+  drawMain: function() {
+    //make buttons and initialize their functions
+    this.startButton = this.makeButton(window.game.world.centerX, window.game.world.centerY - 20, 'START');
+    this.startButton.events.onInputDown.add(function() {window.game.state.start('Cutscene');});
+    this.optionButton = this.makeButton(window.game.world.centerX, window.game.world.centerY + 20, 'OPTIONS');
+    this.optionButton.events.onInputDown.add(this.toggleMenu, this);
+    this.fullscreenButton = this.makeButton(window.game.world.centerX, window.game.world.centerY - 60, 'FULLSCREEN');
+    this.fullscreenButton.visible = false;
+    this.fullscreenButton.events.onInputDown.add(this.fullscreenToggle, this);
+    this.muteButton = this.makeButton(window.game.world.centerX, window.game.world.centerY - 20, 'SOUND: ' + (window.game.sound.mute ? 'OFF':'ON'));
+    this.muteButton.visible = false;
+    this.muteButton.events.onInputDown.add(function(target) {window.game.sound.mute = window.game.sound.mute === false; target.setText('SOUND: '+ (window.game.sound.mute ? 'OFF':'ON'));});
+    this.seedButton = this.makeButton(window.game.world.centerX, window.game.world.centerY + 20, 'SEED: ');
+    this.seedButton.visible = false;
+    this.returnButton = this.makeButton(window.game.world.centerX, window.game.world.centerY + 60, 'RETURN');
+    this.returnButton.visible = false;
+    this.returnButton.events.onInputDown.add(this.toggleMenu, this);
+  },
+
+  //generic button creator, initializes buttons with standard formatting
+  makeButton: function(x, y, name) {
+    var button = window.game.add.text(x, y, name);
+    button.anchor.setTo(0.5, 0.5);
+    button.fontSize = 25;
+    button.fill = 'white';
+    button.inputEnabled = true;
+    button.events.onInputOver.add(function(target) {target.alpha = 150;});
+    button.events.onInputOut.add(function(target) {target.alpha = 1;});
+    return button;
+  },
+
+  //functonality for the fullscreen button
+  fullscreenToggle: function() {
+    if (window.game.scale.isFullScreen)
+    {
+        this.resizeGame(1000, 600);
+        
+        window.game.scale.stopFullScreen();
+    }
+    else
+    {
+        
+       this.resizeGame(window.screen.width, window.screen.height);
+        window.game.scale.startFullScreen();
+        
+        
+    }
+  },
+
+  //toggles between regular menu and options menu
+  toggleMenu: function() {
+    this.startButton.visible = this.startButton.visible === false;
+    this.optionButton.visible = this.optionButton.visible === false;
+    this.fullscreenButton.visible = this.fullscreenButton.visible === false;
+    this.muteButton.visible = this.muteButton.visible === false;
+    this.seedButton.visible = this.seedButton.visible === false;
+    this.returnButton.visible = this.returnButton.visible === false;
+  },
```

Comment:
Clever! It's pretty, but I think it has clarity and scaling issues. 

This kind of only accounts for two states for the main menu, I think using a state machine where a state is kept track of like `OPTIONS` and `MAIN` then using an if statement to show certain buttons would be clearer and scale much better.